<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="messageTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>421</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="typeYourMessageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type your message:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendToEveryoneButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send to everyone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPMButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send PM</string>
    </property>
   </widget>
   <widget class="QListView" name="chatGroupsListView">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chatGroupsLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chatgroups:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentlyOnlineLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Currently online:</string>
    </property>
   </widget>
   <widget class="QLabel" name="extra">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(Choose someone online)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendFileButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send file</string>
    </property>
   </widget>
   <widget class="QLabel" name="messagesLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Messages:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="messagesBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>510</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QListWidget" name="currentlyOnlineListWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="username">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChat_app">
    <property name="title">
     <string>Chat-app</string>
    </property>
   </widget>
   <addaction name="menuChat_app"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
